src/DTO/AddressDTO.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\FanCourierBundle\DTO;
+
+use Webmozart\Assert\Assert;
+
+class AddressDTO
+{
+    public function __construct(
+        public readonly string $locality,
+        public readonly string $county,
+        public readonly string $street,
+        public readonly string $streetNo,
+        public readonly string $zipCode,
+        public readonly string $floor,
```

Comment:
Shouldn't this property be nullable? Otherwise, will it default to an empty string?

Reply:
It is an empty string by default